netci.groovy

```diff
@@ -8,36 +8,46 @@ def branch = GithubBranchName
 // Generate the builds for debug and release, commit and PRJob
 [true, false].each { isPR -> // Defines a closure over true and false, value assigned to isPR
     ['Debug', 'Release'].each { configuration ->
-        // Determine the name for the new job.  The first parameter is the project,
-        // the second parameter is the base name for the job, and the last parameter
-        // is a boolean indicating whether the job will be a PR job.  If true, the
-        // suffix _prtest will be appended.
-        def newJobName = Utilities.getFullJobName(project, configuration, isPR)
-        
-        // Define build string
-        def buildString = """call build.cmd ${configuration}"""
+        ['Windows_NT', 'Ubuntu16.04', 'OSX10.12'].each { osName ->   
+            def isPRString = ""
+            if (isPR)
+            {
+                isPRString = "_prtest"
+            }
+            // Determine the name for the new job.  The first parameter is the project,
+            // the second parameter is the base name for the job, and the last parameter
+            // is a boolean indicating whether the job will be a PR job.  If true, the
+            // suffix _prtest will be appended.
+            def newJobName = "${osName.toLowerCase()}_${configuration.toLowerCase()}${isPRString}"
 
-        // Create a new job with the specified name.  The brace opens a new closure
-        // and calls made within that closure apply to the newly created job.
-        def newJob = job(newJobName) {
+            // Create a new job with the specified name.  The brace opens a new closure
+            // and calls made within that closure apply to the newly created job.
+            def newJob = job(newJobName) {
+                
+                // This opens the set of build steps that will be run.
+                steps {
+                    // Indicates that a batch script should be run with the build string (see above)
+                    // Also available is:
```

Comment:
Comment is a little awkward now that we're using "shell" as well.  Could move explanation of "shell" to above the shell command.